improver/calibration/ensemble_calibration.py

```diff
@@ -966,30 +964,26 @@ def _spatial_domain_match(self):
                 self.current_forecast.coord(axis=axis).points[0],
                 self.current_forecast.coord(axis=axis).points[-1],
             ]
-            if not np.allclose(
-                current_forecast_points, self.coefficients_cube.coord(axis=axis).bounds
-            ):
-                raise ValueError(
-                    msg.format(
-                        axis,
-                        current_forecast_points,
-                        self.coefficients_cube.coord(axis=axis).bounds,
+            for coeff_cube in self.coefficients_cubelist:
+                if not np.allclose(
+                    current_forecast_points, coeff_cube.coord(axis=axis).bounds
+                ):
+                    raise ValueError(
+                        msg.format(
+                            axis,
+                            current_forecast_points,
+                            coeff_cube.coord(axis=axis).bounds,
+                        )
                     )
-                )
 
-    def _calculate_location_parameter_from_mean(self, optimised_coeffs):
+    def _calculate_location_parameter_from_mean(self):
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Method name "_calculate_location_parameter_from_mean" doesn't conform to '((test_[a-zA-Z0-9_]+)|([a-z_][a-z0-9_]{2,30}))$' pattern](https://app.codacy.com/manual/metoppv_tech/improver/pullRequest?prid=5439763)